packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2398,60 +2400,264 @@ test('director notifiers', async t => {
 
   // Not testing rewardPoolAllocation contents because those are simply those values.
   // We could refactor the tests of those allocations to use the data now exposed by a notifier.
+
+  t.pass();
```

Comment:
Isn't it a bug if no assertions get called on `t`?

Reply:
Yeah, this was a holdover from when `assertChange` was asserting without `t`, in which case Ava needn't to be told that if we get to this line we passed. No longer necessary so I'll remove.